CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ In particular, this community seeks the following types of contributions:
 ### Runing OSEM for development
 We are using [Vagrant](https://www.vagrantup.com/) to create our development environments.
 
-1. Install [Vagrant](https://www.vagrantup.com/downloads.html) and [VirtualBox](https://www.virtualbox.org/wiki/Downloads). Both tools support Linux, MacOS and Windows.
+1. Install [Vagrant](https://www.vagrantup.com/downloads.html) and [VirtualBox 5.0.10](https://www.virtualbox.org/wiki/Download_Old_Builds_5_0). Both tools support Linux, MacOS and Windows.
```

Comment:
The installed VirtualBox Guest Additions in the opensuse/openSUSE-42.1-x86_64 image is for the 5.0.10 VirtualBox version. When attempting to use a 5.1.x VirtualBox on OS X, the image failed to provision correctly. Downgrading to 5.0.10 fixed the issue for me.